mmdet/core/hook/data_augment_switch_hook.py

```diff
@@ -0,0 +1,25 @@
+from mmcv.runner.hooks import HOOKS, Hook
+
+
+@HOOKS.register_module()
+class DataAugmentSwitchHook(Hook):
+    """Trun off the mosaic data augment and switch on L1 loss to bbox_head, currently used in YOLOX.
+
+    Args:
+        no_aug_epoch (int): The epoch of close data augmentation. Default to 15.
+    """
+    def __init__(self, no_aug_epoch=15):
+        self.no_aug_epoch = no_aug_epoch
+
+    def before_train_epoch(self, runner):
+        """close mosaic and mixup augmentation and additional L1 loss.
+        """
+        epoch = runner.epoch
+        train_loader = runner.data_loader
+        model = runner.model.module
```

Comment:
please use `is_module_wrapper` in mmcv to avoid this hook being used without DDP or DP